demo-contract-tests/tests/test_fixture/wasm_helper.rs

```diff
@@ -3,33 +3,38 @@ use std::fs;
 use std::path::Path;
 use std::path::PathBuf;
 
-pub fn get_wasm_directory() -> PathBuf {
+pub fn get_wasm_directory() -> (PathBuf, PathBuf) {
```

Comment:
Not a big fan of the return type, one could accidentally index the wrong position and wonder whyt the tests keep failing

Reply:
Maybe an Alias can help here.